app/views/buyers/accounts/_search_results.html.slim

```diff
@@ -37,7 +37,7 @@ table class="data" id="buyer_accounts"
                   { :include_blank => true, :selected => @search.state },
                   { :class => :state }
         th
-          = s.submit 'Search', :name => nil
+          = s.button 'Search', :name => nil, :class => "pf-c-button pf-m-primary"
```

Comment:
```suggestion
          = s.button 'Search', button_html: { class: "pf-c-button pf-m-primary" }
```

Reply:
It doesn't work in this specific form, I don't know why, but I've tried